I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 23, in script
    image arrow_right = image("arrow/arrow_right.png")
  File "game/script.rpy", line 23, in script
    image arrow_right = image("arrow/arrow_right.png")
  File "game/script.rpy", line 23, in <module>
    image arrow_right = image("arrow/arrow_right.png")
NameError: name 'image' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/home/exiel/renpy-7.4.11-sdk/renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "/home/exiel/renpy-7.4.11-sdk/renpy/main.py", line 560, in main
    renpy.game.context().run(node)
  File "game/script.rpy", line 23, in script
    image arrow_right = image("arrow/arrow_right.png")
  File "game/script.rpy", line 23, in script
    image arrow_right = image("arrow/arrow_right.png")
  File "/home/exiel/renpy-7.4.11-sdk/renpy/ast.py", line 1027, in execute
    img = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "/home/exiel/renpy-7.4.11-sdk/renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 23, in <module>
    image arrow_right = image("arrow/arrow_right.png")
NameError: name 'image' is not defined

Linux-5.17.6-1-MANJARO-x86_64-with-glibc2.2.5
Ren'Py 7.4.11.2266
60 Seconde pour Survivre Xavier 1.0
Sat May 21 17:44:46 2022
